src/HotTyper/test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import MovingCursor from './index';
+
+const WrapperStyle = {
```

Comment:
why pascal case for this variable 🤔 ?

Reply:
Sorry, Migrated this from a StyledComponent and forgot the naming 😅